sdk/keyvault/Azure.Security.KeyVault.Certificates/MigrationGuide.md

```diff
@@ -396,3 +399,11 @@ Synchronous methods are also available on `CertificateClient`, though we recomme
 
 - [Key Vault certificates samples for .NET](https://learn.microsoft.com/samples/azure/azure-sdk-for-net/azuresecuritykeyvaultcertificates-samples/)
 - [All Key Vault samples for .NET](https://learn.microsoft.com/samples/browse/?products=azure-key-vault&languages=csharp)
+
+## Support
+
+If you have migrated your code base and are experiencing errors, see our [troubleshooting guide](https://github.com/heaths/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Certificates/TROUBLESHOOTING.md).
```

Comment:
```suggestion
If you have migrated your code base and are experiencing errors, see our [troubleshooting guide](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Certificates/TROUBLESHOOTING.md).
```

I'm assuming the fork pointer was not intended.

Reply:
Busted. 😆